### PR TITLE
Ports some skyrat cooking equipment

### PR DESCRIPTION
**Pull request by MrSamu99**
## About The Pull Request

Ports:
https://github.com/Skyrat-SS13/Skyrat-tg/pull/16903
https://github.com/Skyrat-SS13/Skyrat-tg/pull/16958

## Why It's Good For The Game

You can make alternative variations for food processor and grinder. They use stamina to work and less comfortable to work with, but they're neat.

## Changelog

:cl:
add: Added cutting board
Add: Added big mortar
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+		var/list/choose_options = list(
+			"Grind" = image(icon = 'icons/mob/radial.dmi', icon_state = "radial_grind"),
+			"Juice" = image(icon = 'icons/mob/radial.dmi', icon_state = "radial_juice")
+		)
+		var/picked_option = show_radial_menu(user, src, choose_options, radius = 38, require_near = TRUE)
+
+		if(!length(contents) || !in_range(src, user) || !user.is_holding(attacking_item) && !picked_option)
+			return
+
+		balloon_alert_to_viewers("grinding...")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm]:
```suggestion
		balloon_alert(user, "grinding...")
```

File: code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+		if(!do_after(user, 5 SECONDS, target = src))
+			balloon_alert_to_viewers("stopped grinding")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm]:
```suggestion
			balloon_alert(user, "stopped grinding")
```

File: code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+/obj/item/cutting_board/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	//if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+	//	return
+
+	if(!can_interact(user) || !user.canUseTopic(src, be_close = TRUE))
+		return
+
+	set_anchored(!anchored)
+	balloon_alert_to_viewers(anchored ? "secured" : "unsecured")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm]:
```suggestion
	balloon_alert(user, anchored ? "secured" : "unsecured")
```

File: code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+/obj/structure/large_mortar/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	//if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+	//	return
+
+	if(!can_interact(user) || !user.canUseTopic(src, be_close = TRUE))
+		return
+
+	set_anchored(!anchored)
+	balloon_alert_to_viewers(anchored ? "secured" : "unsecured")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/big_mortar.dm]:
```suggestion
	balloon_alert(user, anchored ? "secured" : "unsecured")
```

File: code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+			balloon_alert(user, "nothing to process")
+			return
+
+		var/datum/food_processor_process/item_process_recipe = GET_RECIPE(contents[1])
+		if(!item_process_recipe)
+			log_admin("DEBUG: [src] (cutting board item) just tried to process [contents[1]] but wasn't able to get a recipe somehow, this should not be able to happen.")
+			return
+
+		//playsound(src, 'sound/effects/butcher.ogg', 50, TRUE)
+		balloon_alert_to_viewers("cutting...")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm:122]:
```suggestion
		balloon_alert(user, "cutting...")
```

File: code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+		if(!do_after(user, 3 SECONDS, target = src))
+			balloon_alert_to_viewers("stopped cutting")
```
Comment (thgvr) [inline code/modules/food_and_drinks/kitchen_machinery/cutting_board.dm]:
```suggestion
			balloon_alert(user, "stopped cutting")
```

**Review by thgvr — COMMENTED** [review]
Hi, just some balloon alert requests. Not sure if it's documented anywhere with them, but generally they should not be shown to everyone around.

**Comment by MrSamu99** [comment]
Well, I fixed all i wanted, so this PR is ready to merge

**Comment by tmtmtl30** [comment]
sure, this is probably fine

**Pull request merged**